Polish Sending Deploys page (#968)
casper-network/docs: source/docs/casper/developers/dapps/sending-deploys.md

# Sending Deploys to a Casper network using the Rust Client

> In order to install smart contracts on the blockchain, you can send your Wasm to the network via a [deploy](../../concepts/design/casper-design.md#execution-semantics-deploys). To do this, you will need to meet a few prerequisites:

Reviewer: Here we want to capitalize Deploy as an object.
```suggestion
To install smart contracts on the blockchain, you can send your Wasm to the network via a [Deploy](../../concepts/design/casper-design.md#execution-semantics-deploys). To do this, you will need to meet a few prerequisites:
```

- You will need a client to interact with the network, such as the [default Casper client](../prerequisites.md#the-casper-command-line-client)
- Ensure you have an [Account](../prerequisites.md#setting-up-an-account) and its associated [keys](../../concepts/accounts-and-keys.md) This account will pay for the Deploy, and its secret key will sign the Deploy
- Ensure this account has enough CSPR tokens to pay for the Deploy

## Paying for Deploys {#paying-for-deploys}

> CSPR tokens are used to pay for Deploys on the Casper Mainnet and Testnet. There are several ways to fund your account:

Reviewer: We capitalize deploys only sometimes. Here's how @ACStoneCL explained it best: "In instances where we're discussing the situation, it might make sense to use deploy. In instances where we're directly referencing it as a portion of code, Deploy might be better". 
```suggestion
CSPR tokens are used to pay for deploys on the Casper Mainnet and Testnet. There are several ways to fund your account:
```

- You may want to [transfer tokens from an exchange](../../users/funding-from-exchanges.md)
- You can use a [block explorer to transfer tokens](../../users/token-transfer.md) between accounts' purses
- You can also [transfer tokens using the default Casper client](../cli/transfers/index.md)
- On the Testnet, you can use the [faucet functionality](../../users/testnet-faucet.md) for testing your smart contracts

> - When running a Network locally, using [nctl](../../dapp-dev-guide/building-dapps/nctl-test.md), you have several funded accounts to play with

Reviewer: Please use a more formal voice:
```suggestion
- If running a network locally using [NCTL](../../dapp-dev-guide/building-dapps/nctl-test.md), the tool provides several funded accounts
```

4. `payment-amount` - The payment for the Deploy in motes. This example uses 2.5 CSPR, but you need to modify this for your contract. See the [note](#a-note-about-gas-price) below
5. `session-path` - The path to the contract Wasm, which should point to wherever you compiled the contract (.wasm file) on your computer

> Once you execute this command, it will return a Deploy hash, which you will need to verify that the Deploy was accepted by the node and to query the execution results of the Deploy. Sending a Deploy to the network does not mean that the Deploy was processed successfully. Once the network has received the Deploy and done some preliminary validation of it, it will queue up in the system before being proposed in a block for execution. Therefore, you will need to check to see that the Deploy was executed as expected.

Reviewer: This is one of those places where I wouldn't capitalize deploy, since we're discussing the situation.
Also, I would say "The command will return ..." to improve the original text. We overuse the word "execute" in our docs.

> Once you execute this command, it will return a Deploy hash, which you will need to verify that the Deploy was accepted by the node and to query the execution results of the Deploy. Sending a Deploy to the network does not mean that the Deploy was processed successfully. Once the network has received the Deploy and done some preliminary validation of it, it will queue up in the system before being proposed in a block for execution. Therefore, you will need to check to see that the Deploy was executed as expected.

Reviewer: I think my previous comment was missed. I would prefer this simpler explanation:
```suggestion
The command will return a deploy hash, which is needed to verify the deploy's execution results. Sending the deploy and receiving the deploy hash does not mean the deploy was processed successfully. Therefore, you must check the deploy execution using the deploy hash. See the deploy lifecycle for more details.
```
Of course, it would be good to link to the deploy lifecycle, but that page is changing location. It is currently here: /concepts/design/casper-design/#execution-semantics-phases.
